test/e2e/cypress/integration/profiles/code/login/error.spec.ts

```diff
@@ -125,7 +125,7 @@ context("Login error messages with code method", () => {
         )
       })
 
-      it("should show error message when required fields are missing", () => {
+      it.only("should show error message when required fields are missing", () => {
```

Comment:
Won't that `only` here prevent other tests from being executed? Or is it actually the intent here?

Reply:
Good point, cypress unfortunately has no tooling around preventing this in CI. :/

```suggestion
      it("should show error message when required fields are missing", () => {
```